sqrl-planner/src/main/java/com/datasqrl/graphql/inference/GraphqlModelGenerator.java

```diff
@@ -158,6 +187,15 @@ protected void walkMutation(APISource source, TypeDefinitionRegistry registry,
     mutations.add(mutationCoords);
   }
 
+  private Optional<EnginePhysicalPlan> getLogPlan() {
+    //Todo: Bad instance checking, Fix to bring multiple log engines (?)
```

Comment:
there are a lot of assumptions of a single log engine in the vertx server too. we can fix this down the road when we introduce calcite as a query delegation layer.